test/sdk/rust/src/main.rs

```diff
@@ -112,6 +112,69 @@ fn run() -> Result<(), String> {
     sdk.set_label("test-label", &creation_ts.to_string())
         .map_err(|e| format!("Could not run SetLabel(): {}. Exiting!", e))?;
 
+    println!("Setting player capacity...");
```

Comment:
We could do a simple string comparison to see if "PlayerTracking=true" is contained in the FEATURE_GATES environment variable as a way to determine whether to run these tests or not?

Keep things simple, WDYT?

Reply:
OK. It seems easy to implement so I'll fix.

Reply:
I fixed it.